drivers/clock_control/Kconfig

```diff
@@ -1,13 +1,14 @@
 #
-# Copyright (c) 2018 Nordic Semiconductor ASA
+# Copyright (c) 2018 - 2019 Nordic Semiconductor ASA
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-config CLOCK_CONTROL_NRF_LL_NRFXLIB
-	bool "nRF BLE Controller clock control driver"
+config CLOCK_CONTROL_NRFXLIB
+	bool "Enable nrfxlib clock control driver, for use with BLE Controller or MPSL clock control."
 	default y
 	select CLOCK_CONTROL_NRF_FORCE_ALT
+  depends on MULTITHREADING_LOCK
```

Comment:
Wrong indentation.
And shouldn't it be rather `select`? How is `MULTITHREADING_LOCK` supposed to be enabled?